pkgs/games/leela-zero/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, fetchFromGitHub, cmake, boost, eigen
+, opencl-headers, ocl-icd, qtbase , zlib }:
+
+stdenv.mkDerivation rec {
+  name = "leela-zero-${version}";
+  version = "0.16";
+
+  src = fetchFromGitHub {
+    owner = "gcp";
+    repo = "leela-zero";
+    rev = "4fd6e694ead45b7cbb14ea76c5bdfcbfa662304b";
+    sha256 = "1px7wqvlv414gklzgrmppp8wzc2mkskinm1p75j4snbqr8qpbn5s";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [ boost opencl-headers ocl-icd qtbase zlib ];
+
+  nativeBuildInputs = [ cmake ];
+
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
+    description = "Go engine modeled after AlphaGo Zero";
+    homepage    = https://github.com/gcp/leela-zero;
+    license     = licenses.gpl3;
+    maintainers = maintainers.averelld;
```

Comment:
This must be a list

```suggestion
    maintainers = [ maintainers.averelld ];
```

Reply:
Fixed. This doesn't give any errors when doing nixos-rebuild, how can I force evaluation of this section?

Reply:
Use this:
```
nix-build --no-out-link /path/to/nixpkgs-repo -A leela-zero
```
A more general snippet to test a new package is:
```
nix-build --no-out-link -E '(import /path/to/nixpkgs-repo { config = {}; overlays = []; }).leela-zero'
```
This ignores your local config, so you get the same package result as with Hydra (the Nix build server).